website/util/__init__.py

```diff
@@ -58,7 +57,7 @@ def api_url_for(view_name, _absolute=False, _xml=False, *args, **kwargs):
 def api_v2_url(path_str,
                params=None,
                base_route=website_settings.API_DOMAIN,
-               base_prefix=api_settings.API_PREFIX,
+               base_prefix='',
```

Comment:
In the process of throwing out API_PREFIX, we accidentally dropped `API_BASE` as well. (and since the tests were deployment-independent, it wasn't obvious)  

I think that was an accident (refer to `api/base/urls.py`) and will submit another PR to put that back.